pkg/minishift/openshift/oc.go

```diff
@@ -68,3 +90,114 @@ func AddContextForProfile(profile string, ip string, username string, namespace
 	}
 	return nil
 }
+
+// Get the route for service
+func GetServiceURL(service, namespace string, https bool) (string, error) {
+	urlScheme := "http://"
+	if https {
+		urlScheme = "https://"
+	}
+
+	if namespace == "default" {
```

Comment:
So why is it not allowed to get a service from the 'default' namespace? 'default' is not == no or empty namespace.